cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaBridgeCluster.java

```diff
@@ -508,10 +550,101 @@ protected void setBootstrapServers(String bootstrapServers) {
         this.bootstrapServers = bootstrapServers;
     }
 
+
+    protected void setRack(Rack rack) {
+        this.rack = rack;
+    }
+
+    protected void setInitImage(String initImage) {
+        this.initImage = initImage;
+    }
+
     public KafkaBridgeHttpConfig getHttp() {
         return this.http;
     }
 
+    public String getInitContainerClusterRoleBindingName() {
+        return KafkaBridgeResources.initContainerClusterRoleBindingName(cluster, namespace);
+    }
```

Comment:
Can't the place which is calling this call `KafkaBridgeResources.initContainerClusterRoleBindingName` directly? We tried to avoid these _call hierarchies_ lately.